src/cloud/env/variables-post-deploy.md

```diff
@@ -131,6 +131,19 @@ Customize the list of pages used to preload the cache in the `post_deploy` stage
                - "store-page:/contact-us:code1|code2"
    ```
 
+### `WARM_UP_CONCURRENCY`
```

Comment:
I advise putting variables in alphabetic order. As the list grows, this becomes important.

Reply:
That's a good idea. @dhorytskyi  Can you move the WARM_UP_CONCURRENCY variable description before `WARM_UP_PAGES`.

Reply:
Done